lms/static/sass/base/_variables.scss

```diff
@@ -330,6 +330,7 @@ $error-color: $error-red;
 $warning-color: $m-pink;
 $confirm-color: $m-green;
 $active-color: $blue;
+$highlight-color: rgb(255,255,0);
```

Comment:
NOTE: adding default highlight color to variables (to highlight any user queries/searches)